ada/extensions/nodes/ada_node/bodies

```diff
@@ -55,6 +55,8 @@ function Ada_Node_P_Filter_Is_Imported_By
       Hash            => Hash,
       Equivalent_Keys => "=");
 
+   type CU_Array is array (Positive range <>) of Bare_Compilation_Unit;
```

Comment:
Maybe not important for now, but wondering if `All_Compilation_Units` shouldn't be defined as a regular property